.github/ISSUE_TEMPLATE/15_bug-report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Bug report
+about: Report something broken in an Altinity Stable Build 
+title: ''
+labels: 'potential bug'
+assignees: ''
+
+---
+
+✅  *I checked [the Altinity Stable Builds lifecycle table](https://docs.altinity.com/altinitystablebuilds/#altinity-stable-builds-life-cycle-table), and the Altinity Stable Build version I'm using is still supported.*
+
+## Type of problem
+Choose one of the following items, then delete the others: 
+
+**Bug report** - something's broken
+
+**Incomplete implementation** - something's not quite right
+
+**Performance issue** - something works, just not as quickly as it should
+
+**Backwards compatibility issue** - something used to work, but now it doesn't
+
+**Unexpected behavior** - something surprising happened, but it wasn't the good kind of surprise
+
+**Installation issue** - something doesn't install they way it should
+
+**Usability issue** - something works, but it could be a lot easier
+
+**Documentation issue** - something in the docs is wrong, incomplete, or confusing
+
+## Describe the situation
+A clear, concise description of what's happening. 
+
```

Comment:
Something to add when discussion reproduction:  Does it reproduce in a ClickHouse Official build of the same version?

Reply:
Can we also request a minimum viable scenario/reproduction script?